lib/index.js

```diff
@@ -107,7 +107,7 @@ function renderFile (root, obj) {
 }
 
 function renderUrl (obj) {
-  return rest(obj.url).then((res) => { return JSON.parse(res.entity) })
+  return rest(obj.url).then((res) => { return JSON.parse(transformRaw(obj, res.entity)) })
```

Comment:
I could add this to the renderFile, renderData, and renderGraphql. Do you think this would be beneficial? I'm assuming that with a File or Data you could feed in a proper JSON format.